jcl/src/java.base/share/classes/java/lang/VMAccess.java

```diff
@@ -174,4 +182,25 @@ public Package getSystemPackage(String name) {
 		return Package.getSystemPackage(name);
 	}
 	/*[ENDIF]*/
+
+	/**
+	 * Returns a J9ClassRef object.
+	 * 
+	 * @param addr - the native addr of the J9Class
+	 * @return A J9Class reference object
```

Comment:
Above it says J9ClassRef and here it says J9Class, although it returns an InternalRamClass